examples/web-wagmi/src/profiles/components/ProfileCard.tsx

```diff
@@ -10,9 +9,7 @@ type ProfileCardProps = {
 export function ProfileCard({ profile }: ProfileCardProps) {
   return (
     <article>
-      <Link to={`/profile/handle/${profile.handle}`}>
-        <ProfilePicture picture={profile.picture} />
-      </Link>
+      <ProfilePicture picture={profile.picture} />
```

Comment:
There is no support, for now, to provide route `param` with the handle so it was pointing to a nonexisting route.